### PR TITLE
[python] amend revision question

### PR DESCRIPTION
**Pull request by emmbow**
Based on Nem's feedback:

```I think that we should remove this question.
Don't see how it's possible to accurately estimate this but more importantly I don't think this information is particularly useful.
Although it could be true that for basic things you  write less code with Python than with CPP, it's definitely not always true.```

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/core/meet-python/why-learn-python.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/python/core/meet-python/why-learn-python.md?ref=3744979b51f621d6311cea3a0b4cd9753e78c46a)

File: python/core/meet-python/why-learn-python.md [hunk 1]
```diff
@@ -58,12 +58,12 @@ It typically requires ??? code to write the same task in Python than in C++.
...
+Python is the preferred language for ???.
 
-- `Up to 10 times less`
-- `Up to 5 times less`
-- `3–5 times less`
-- `Roughly equal`
+- data science
+- making games
+- web design
+- APIs
```
Comment (kapnobatai137) [inline python/core/meet-python/why-learn-python.md:66]: I think the question is still subjective as people can have different opinions depending on their background/domain they are working in

maybe we can make this question into a discussion insight? something like: "What will you use Python for? Discuss in the comments"

cc @nem035

Comment (nem035) [reply python/core/meet-python/why-learn-python.md:66]: Yeah, definitely still subjective although this is an improvement.

Let's start a thread about this in our Slack?

**Comment by kapnobatai137** [comment]
old PR, content was updated in the meantime

**Pull request closed without merging**